index.bs

```diff
@@ -358,324 +493,120 @@ For the main <dfn>sanitize</dfn> operation, using a {{ParentNode}} |node|, a
-       1. Set |result|["{{SanitizerConfig/attributes}}"] to the
-          [=set/difference=] of |default|["{{SanitizerConfig/attributes}}"]
-          and |result|["{{SanitizerConfig/removeAttributes}}"].
-       1. [=set/Remove=] "{{SanitizerConfig/removeAttributes}}" from |result|.
-   1. If neither |config|["{{SanitizerConfig/attributes}}"] nor
-      |config|["{{SanitizerConfig/removeAttributes}}"] [=map/exist=]:
-      1. Set |result|["{{SanitizerConfig/attributes}}"] to
-         |default|["{{SanitizerConfig/attributes}}"].
-1. Else (if not |safe|):
-   1. If neither  |config|["{{SanitizerConfig/elements}}"] nor
-      |config|["{{SanitizerConfig/removeElements}}"] [=map/exist=]:
-      1. Set |result|["{{SanitizerConfig/elements}}"] to
-         |default|["{{SanitizerConfig/elements}}"].
-   1. If neither  |config|["{{SanitizerConfig/attributes}}"] nor
-      |config|["{{SanitizerConfig/removeAttributes}}"] [=map/exist=]:
-      1. Set |result|["{{SanitizerConfig/attributes}}"] to
-         |default|["{{SanitizerConfig/attributes}}"].
-1. [=Assert=]: |result| is [=SanitizerConfig/valid=].
-1. [=Assert=]: |result| is [=SanitizerConfig/canonical=].
+Note: While this algorithm is called [=remove unsafe=], we use
+    <a href="#security-considerations">the term "unsafe" strictly in the sense
+    of this spec</a>, to denote content that will
+    execute JavaScript when inserted into the document. In other words, this
+    method will remove oportunities for XSS.
+
+To <dfn for="SanitizerConfig">remove unsafe</dfn> from a |config|, do this:
+
+1. [=Assert=]: The [=built-in safe baseline config=] has
+   {{SanitizerConfig/removeElements}} and {{SanitizerConfig/removeAttributes}}
+   keys set, but not {{SanitizerConfig/elements}},
+   {{SanitizerConfig/replaceWithChildrenElements}}, or
+   {{SanitizerConfig/attributes}}.
+1. Let |result| be a copy of |config|.
+1. [=list/For each=] |elem| in
+   [=built-in safe baseline config=][{{SanitizerConfig/removeElements}}]:
+    1. Call |result|.{{Sanitizer/removeElement()|removeElement}}(|elem|)
+1. [=list/For each=] |attr| in
+   [=built-in safe baseline config=][{{SanitizerConfig/removeAttributes}}]:
+    1. Call |result|.{{Sanitizer/removeAttribute()|removeAttribute}}(|attr|)
 1. Return |result|.
 
 </div>
 
 <div algorithm>
-In order to <dfn>canonicalize a sanitizer element list</dfn> |list|, with a
-default namespace |defaultNamespace|, run the following steps:
-
-1. Let |result| be a new [=ordered set=].
-2. [=list/iterate|For each=] |name| in |list|, call
-   [=canonicalize a sanitizer name=] on |name| with |defaultNamespace| and
-   [=set/append=] to |result|.
-3. Return |result|.
+To <dfn for="Sanitizer">set a config</dfn> |config| on a {{Sanitizer}} |sanitizer|, do this:
+
+1. [=Assert=]: |config| is a [=dictionary=].
+1. [=list/iterate|For each=] |item| of |config|[{{SanitizerConfig/elements}}] do:
+  1. Call |sanitizer|.{{Sanitizer/allowElement()|allowElement}}(|item|).
```

Comment:
Same problem as above. You cannot call public API here.

Reply:
Done.